opba-db/src/test/java/de/adorsys/opba/db/BankProtocolActionsSqlGeneratorTest.java

```diff
@@ -0,0 +1,153 @@
+    private static final String BANK_SUB_ACTION_CSV_HEADER = "id,action_id,protocol_action,sub_protocol_bean_name";
+    private static final String BANK_PROFILE_CSV_HEADER = "uuid,name,bic,url,adapter_id,bank_code,idp_url,aspsp_sca_approaches";
+
+    @Value("${bank-action-generator.action.start-id}")
+    private Integer bankActionId;
+
+    @Value("${bank-action-generator.sub-action.start-id}")
+    private Integer bankSubActionId;
+
+    @Value("${bank-action-generator.skip-banks}")
+    private Integer skipBanks;
+
+    @Test
+    @SneakyThrows
+    public void convertToDbSql() {
+        List<String> banks = skipBanksFromFileBeginning(readResourceLines(BANK_DATA_SOURCE_PATH));
+        prepareDestinationFiles();
+
+        for (String bank : banks) {
+            writeXs2aBankActionData(bank);
+            writeHbciBankData(bank);
+        }
+    }
+
+    private List<String> skipBanksFromFileBeginning(List<String> banks) {
+        return banks.subList(skipBanks, banks.size());
+    }
+
+    private void writeXs2aBankActionData(String bankRecord) {
+        String bankUUID = bankRecord.substring(0, bankRecord.indexOf(','));
+        int authorizationId;
+
+        writelnToFile(BANK_ACTION_DESTINATION_PATH, String.format("%d,%s,LIST_ACCOUNTS,xs2aListAccounts,true", bankActionId++, bankUUID));
+        writelnToFile(BANK_ACTION_DESTINATION_PATH, String.format("%d,%s,LIST_TRANSACTIONS,xs2aSandboxListTransactions,true", bankActionId++, bankUUID));
+        writelnToFile(BANK_ACTION_DESTINATION_PATH, String.format("%d,%s,AUTHORIZATION,,true", authorizationId = bankActionId++, bankUUID));
+        writelnToFile(BANK_ACTION_DESTINATION_PATH, String.format("%d,%s,SINGLE_PAYMENT,xs2aInitiateSinglePayment,true", bankActionId++, bankUUID));
+        writelnToFile(BANK_ACTION_DESTINATION_PATH, String.format("%d,%s,GET_PAYMENT_INFORMATION,xs2aGetPaymentInfoState,true", bankActionId++, bankUUID));
+        writelnToFile(BANK_ACTION_DESTINATION_PATH, String.format("%d,%s,GET_PAYMENT_STATUS,xs2aGetPaymentStatusState,true", bankActionId++, bankUUID));
+
+        writelnToFile(BANK_SUB_ACTION_DESTINATION_PATH, String.format("%d,%d,GET_AUTHORIZATION_STATE,xs2aGetAuthorizationState", bankSubActionId++, authorizationId));
+        writelnToFile(BANK_SUB_ACTION_DESTINATION_PATH, String.format("%d,%d,UPDATE_AUTHORIZATION,xs2aUpdateAuthorization", bankSubActionId++, authorizationId));
+        writelnToFile(BANK_SUB_ACTION_DESTINATION_PATH, String.format("%d,%d,FROM_ASPSP_REDIRECT,xs2aFromAspspRedirect", bankSubActionId++, authorizationId));
+        writelnToFile(BANK_SUB_ACTION_DESTINATION_PATH, String.format("%d,%d,DENY_AUTHORIZATION,xs2aDenyAuthorization", bankSubActionId++, authorizationId));
+    }
+
+    private void writeHbciBankData(String bankRecord) {
+        String recordWithNewUUID = replaceWithRandomUUID(bankRecord);
+        writelnToFile(BANK_PROFILE_DESTINATION_PATH, recordWithNewUUID);
```

Comment:
For HBCI one should use `hbci-adapter` in bank_profile `adapter_id` - currently i.e. verlag-adapter is for `00195358-fbf1-4a33-8097-d6d7e6e338cd` but its actions are HBIC

Reply:
Added 'HBCI' prefix for the bank name and 'hbci-' prefix for the adapter_id